lib/diagnostic.js

```diff
@@ -8,21 +8,40 @@
 // and returns the minimum from that set
 // assume that all the argument are numbers
 const min = function min() {
+  let lowNum = arguments[0];
 
+  for (let i = 0; i < arguments.length; i--) {
```

Comment:
This decrements ```i``` after each iteration so that it'll always be less than ```arguments.length``` if any arguments are passed into the function, causing this to become an infinite loop.